Jenkinsfile.SensorboxTests

```diff
@@ -1,17 +1,50 @@
 #!groovy
 pipeline {
-        agent {
-                node {
-                    label 'HardwareSensorBox'
-              }
-         }
+     agent {
+        dockerfile {
+            filename 'Dockerfile.jenkins'
+            // 'docker build' would normally copy the whole build-dir to the container, changing the
+            // docker build directory avoids that overhead
+            dir 'docker'
+            // run on slave1 where nexus is connected to
+            label 'DeviceHardware'
```

Comment:
The test run here are specific to the sensor box. Thus it should be mandatory.
In the future more devices, including those with robot arm, will be added.
Thus I think we should be specific here and require the HardwareSensorBox.